carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/locator/ExtendedFindByBuilder.java

```diff
@@ -0,0 +1,68 @@
+package com.qaprosoft.carina.core.foundation.webdriver.locator;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.AbstractFindByBuilder;
+
+import com.qaprosoft.carina.core.foundation.commons.SpecialKeywords;
+import com.sun.jersey.core.util.Base64;
+
+import io.appium.java_client.MobileBy;
+
+public abstract class ExtendedFindByBuilder extends AbstractFindByBuilder {
+    
+    private static final Logger LOGGER = Logger.getLogger(ExtendedFindByBuilder.class);
+    
+    private static final String BASE_IMAGE_DIR = System.getProperty("user.dir") + "/src/test/resources".replaceAll("/", File.separator);
```

Comment:
@vdelendik What do you think about base directory for images? I was thinking about that and suggest to change default storage to /src/MAIN/resources